app/src/main/java/org/wikiedufoundation/wikiedudashboard/ui/campaign/dao/ActiveCampaignDao.kt

```diff
@@ -0,0 +1,31 @@
+package org.wikiedufoundation.wikiedudashboard.ui.campaign.dao
+
+import androidx.lifecycle.LiveData
+import androidx.room.Dao
+import androidx.room.Insert
+import androidx.room.OnConflictStrategy
+import androidx.room.Query
+import org.wikiedufoundation.wikiedudashboard.ui.campaign.data.CampaignListData
+
+/**
+ * This creates an interface for
+ * querying the database
+ **/
+@Dao
+interface ActiveCampaignDao {
+
+    /**
+     * This get all the campaigns
+     *
+     **/
+    @Query("SELECT * from campaign_list ORDER BY id ASC")
+    fun getAllCampaign(): LiveData<List<CampaignListData>>
+
+    /**
+     * This insert the data into the database
+     *
+     **/
+    @Insert(onConflict = OnConflictStrategy.IGNORE)
```

Comment:
@victorvicari suggested to use REPLACE instead of IGNORE I think.

Reply:
Replace was not a good option here as I removed the autocomplete = true from the primary key. Any duplication will be from the server